README.md

```diff
@@ -47,19 +47,25 @@ More detailed build instructions can be found [here](docs/install_vortex.md).
 - [Yosys](https://github.com/YosysHQ/yosys)
 - [Sv2v](https://github.com/zachjs/sv2v)
 ### Install development tools
-    $ sudo apt-get install build-essential
-    $ sudo apt-get install binutils
-    $ sudo apt-get install python
-    $ sudo apt-get install uuid-dev
-    $ sudo apt-get install git
+    $ sudo apt-get install build-essential git binutils python3 uuid-dev wget ocl-icd-opencl-dev
```

Comment:
Was `wget` not installed by default on Ubuntu?
Also for which part of Vortex was the missing `ocl-icd-opencl-dev` package an issue?

Reply:
not on a fresh docker image, perhaps user may have it but I think build instruction should get user from fresh docker image to fully working

```bash
~ docker run -it ubuntu:22.04 /bin/bash

root@6fbb29d4c97d:/# wget
bash: wget: command not found
```
I think I had issues running the hello world vecAdd kernel without ocl-icd-opencl-dev

Reply:
I'm actually interested in this fix to support more OS.
can you remove the runtime stuff and only submit this update standalone with configure and README.md?